### PR TITLE
BAQE-1692 - kogito-apps integration tests initial version

### PR DESCRIPTION
**Pull request by hellowdan**

### CODE REVIEW

**Review by jstastny-cz — COMMENTED** [review]
Looks good, if there are some failing modules, please put them as excluded, we can investigate later.

File: run-tests-with-build-kogito-apps/pom.xml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    <artifactId>run-tests-with-build</artifactId>
+    <groupId>org.kie</groupId>
+    <version>1.0-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>run-tests-with-build-kogito-apps</artifactId>
+
+  <properties>
+    <!-- integration-tests-quarkus-decisions tests depends on src/main/java classes -->
```
Comment (jomarko) [inline run-tests-with-build-kogito-apps/pom.xml:15]: similar comment as for `kogito-examples`, if we know some example where is it needed, or some general formulation

File: run-tests-with-build-kogito-apps/pom.xml [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-invoker-plugin</artifactId>
+        <configuration>
+          <pomIncludes>
+            <pomInclude>kogito-apps*/apps-integration-tests/integration-tests-jobs-service/*/pom.xml</pomInclude>
+            <pomInclude>kogito-apps*/apps-integration-tests/integration-tests-trusty-service/*/pom.xml</pomInclude>
```
Comment (jomarko) [inline run-tests-with-build-kogito-apps/pom.xml:57]: I am fine with this as initial version, in following PRs we can add modules with tests
- `find -name *Test.java`
- `find -name *IT.java`

File: run-tests-with-build-kogito-apps/pom.xml [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+                </artifactItem>
+              </artifactItems>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-invoker-plugin</artifactId>
+        <configuration>
```
Comment (jomarko) [inline run-tests-with-build-kogito-apps/pom.xml:54]: If there will be failures later about addresses in use, we can solve using `<parallelThreads>1</parallelThreads>`. just FYI. no update needed right now.

**Review by jomarko — COMMENTED** [review]
One comment to address, two just information to share. Thank you @hellowdan

**Review by jstastny-cz — APPROVED** [review]

File: run-tests-with-build-kogito-apps/pom.xml (hunk 2 above)
Comment (hellowdan) [reply run-tests-with-build-kogito-apps/pom.xml]: update sent :)

**Review by jomarko — APPROVED** [review]
Thank you @hellowdan . The list of modules seems to be complete to me.

**Comment by jstastny-cz** [comment]
Seeing one failing module in my local, running in CI.
```
[ERROR] The following builds failed:
[ERROR] *  kogito-apps/explainability/explainability-integrationtests/explainability-integrationtests-pmml/pom.xml
```

**Comment by hellowdan** [comment]
> Seeing one failing module in my local, running in CI.
> 
> ```
> [ERROR] The following builds failed:
> [ERROR] *  kogito-apps/explainability/explainability-integrationtests/explainability-integrationtests-pmml/pom.xml
> ```

It looks like this one is unstable. I categorized based on the last successful build, but now I remember that it failed once.

File: run-tests-with-build-kogito-apps/pom.xml (hunk 3 above)
Comment (jstastny-cz) [reply run-tests-with-build-kogito-apps/pom.xml:53]: It was actually required when we added some of the excluded modules. So it's already safe.

**Review by jstastny-cz — APPROVED** [review]
LGTM.

**Pull request merged**